docs/docs/deploy-action-server.mdx

```diff
@@ -0,0 +1,379 @@
+---
+id: deploy-action-server
+sidebar_label: Deploy Action Server
+title: Deploy Rasa Action Server
+abstract: This page explains how to build an Action Server Image and deploy a Rasa Action Server using Helm. The second of three steps in deploying your Rasa assistant.
+---
+<!-- this file is version specific, do not use `@site/...` syntax -->
+import variables from './variables.json';
+
+:::note
+The [Rasa Action Server chart](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa-action-server) is open source and available in the
+[helm-charts repository](https://github.com/rasahq/helm-charts).
+Please
+[create an issue](https://github.com/RasaHQ/helm-charts/issues/new) in this
+repository if you discover bugs or have suggestions for improvements.
+
+:::
+
```

Comment:
Can we add an overview of what you will do here? This a large file, but it largely boils down to install the Rasa Action Server helm chart and then connect it to your Rasa Open Source deployment